packages/mobile/src/features/videos/components/MainScreen/BigHeaderButton.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FontWeight } from '../../../common/styles/constans/FontWeight';
+import { FontSize } from '../../../common/styles/constans/FontSize';
+import { Color } from '../../../common/styles/constans/Color';
+import { MyText } from '../../../common/styles/MyText';
+
+const Wrapper = styled.TouchableOpacity`
```

Comment:
jeśli w rzaden sposób nie stylujesz to nie ma sensu tworzyć odzielnej klasy, tylko uyj tam TouchableOpacity

Reply:
zrobione